client/webpack.config.js

```diff
@@ -1,41 +1,23 @@
 const path = require('path')
-const MiniCssExtractPlugin = require('mini-css-extract-plugin')
 
 module.exports = {
-  entry: ['./client/index.js', './client/styles/index.scss'],
+  entry: ['./client/index.js'],
   output: {
     path: path.join(__dirname, '..', 'server', 'public'),
-    filename: 'bundle.js'
+    filename: 'bundle.js',
   },
   mode: 'development',
-  plugins: [
```

Comment:
does this mean we're not bundling our css anymore? was that an intended change

Reply:
I've snooped a couple of other repos (`sweet-as-beers` and `react-to-web-api`) and neither of them have this `MiniCssExtractPlugin` stuff. I've never heard of it so presumably neither have our students. Let's keep it simple. So I'm retroactively making this an intentional change! :-)

Reply:
okay cool